homeassistant/components/alarm_control_panel/__init__.py

```diff
@@ -146,6 +184,14 @@ def alarm_trigger(self, code=None):
         """Send alarm trigger command."""
         raise NotImplementedError()
 
+    def alarm_keypress(self, keypress=None):
+        """Send custom key sequence to alarm."""
+        raise NotImplementedError()
```

Comment:
Are you sure this should be a required method for all alarms? I suggest replacing this with `pass`.